templates/_layouts/docs.html

```diff
@@ -32,7 +32,7 @@
               {% macro side_nav_item(url, title) -%}
                   <li class="p-side-navigation__item">
                     <a class="p-side-navigation__link" href="{{ url }}" {% if (slug and slug in url) or (url == path) %}aria-current="page"{% endif %}>
-                      {{ title }}
+                      {{ title | safe | replace("%version", version) }}
```

Comment:
Is `safe` needed? I don't think we expect any HTML in titles, so I think it should be actually safer without `safe` (so any accidental HTML will be encoded).

Also I think I'd prefer the interpolation to look more like a variable in template or Python interpolation, so maybe use `{version}` instead of `%version` please.